pages/DSNP/Identity.md

```diff
@@ -26,6 +26,22 @@ the ability to delegate permission to others to announce content on the user's b
 * Announcements from a delegate MUST be able to be verified as to which delegate made the specific announcement.
 * Delegation revocation MUST NOT be retroactive.
 
+## Operations
+
+Each of the these MUST have verifiable approval of the acting
+principal(s) via signature proof or other system. For example, an
+implementation might use the public key of an asymmetric key pair as a
+control key, and provide a proof for each operation by producing a
+cryptographic signature of the inputs and some nonce value.
+
+| Operation | Optional? | Principal(s) | Inputs | Outputs/State Changes |
+|---------- |---------- |------------- |------- |-------------- |
+| Create DSNP Identifier | no | User | None | New DSNP Identifier connected to User's Control Key |
```

Comment:
We don't call it a DSNP Delegation, should we shorten this to just Create Identifier?
```suggestion
| Create Identifier | no | User | None | New DSNP Identifier connected to User's Control Key |
```

Reply:
User's control key should be an input here right? Although perhaps that is implied by the User as Principal? We don't require it on `Add Control Key`, but the same could apply there.
```suggestion
| Create DSNP Identifier | no | User | Control Key, Proof of Ownership of Control Key | New DSNP Identifier connected to User's Control Key |
```

Reply:
> We don't call it a DSNP Delegation, should we shorten this to just Create Identifier?

That certainly makes things more concise, and since that's the only type of identifier we're interested in, I think we can change "DSNP Identifier" to "Identifier" throughout the operation spec sections.

> User's control key should be an input here right? Although perhaps that is implied by the User as Principal? We don't require it on Add Control Key, but the same could apply there.

I agree, I think we can be explicit here that the control key (and proof) are parameters, since no user authentication context exists yet at the time of operation invocation.